### PR TITLE
release v2.1.1

### LINKED ISSUE #563: Make release with latest translation-lib
Should happen before 1.21 release.

https://github.com/kubernetes/kubernetes/pull/97823 has the changes we want.

/cc @Jiawei0227

### PR DESCRIPTION
**Pull request by pohly**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

We have accumulated several bug fixes for v2.1.0 which would be good to have in a patch release.

**Which issue(s) this PR fixes**:
Fixes #563

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by pohly** [comment]
/hold

We first need to agree whether we have everything for the release, see https://github.com/kubernetes-csi/external-provisioner/issues/563#issuecomment-795157652

File: CHANGELOG/CHANGELOG-2.1.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.1.0
+
+## Changes by Kind
+
+### Bug or Regression
+ - External-provisioner may have [stopped provisioning](https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/103) for a PVC depending on certain timing conditions (provisioning failed once, next attempt currently running, PVC update arrives from API server). ([#564](https://github.com/kubernetes-csi/external-provisioner/pull/564), [@pohly](https://github.com/pohly))
+ - Producing storage capacity may have failed with `the object has been modified` errors ([#568](https://github.com/kubernetes-csi/external-provisioner/pull/568), [@pohly](https://github.com/pohly))
+ - `k8s.io/csi-translation-lib` dependency is updated from v0.20.0 to v0.20.4 to adopt fixes in the upstream library. ([#571](https://github.com/kubernetes-csi/external-provisioner/pull/571), [@ialidzhikov](https://github.com/ialidzhikov))
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-2.1.md:12]: sorry I just updated the release note for this PR since it's a obsolete now. Could you refresh again?

Comment (pohly) [reply CHANGELOG/CHANGELOG-2.1.md]: Done.

**Pull request merged**